examples/general/non_local_effect_before_error_return/ui/main.stderr

```diff
@@ -12,7 +12,7 @@ LL |     Err(VarError::NotPresent)
    = note: `-D non-local-effect-before-error-return` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(non_local_effect_before_error_return)]`
 
-error: call with mutable reference before error return
+error: call to std::vec::Vec::<u32>::push with mutable reference before error return
```

Comment:
```suggestion
error: call to `std::vec::Vec::<u32>::push` with mutable reference before error return
```
Since the function paths are technically syntax, could we enclose them in backticks (as shown)?